tests/lib/rules/prefer-wait-for.test.ts

```diff
@@ -218,6 +218,21 @@ ruleTester.run(RULE_NAME, rule, {
       cy.wait();
       `,
     },
+    {
+      settings: {
+        'testing-library/utils-module': 'test-utils',
+      },
+      code: `
+      // case: aggressive reporting disabled - method named same as invalid method
+      // but not coming from Testing Library is valid
+      import { wait as testingLibraryWait } from 'test-utils'
```

Comment:
Adding an extra test to this rule since was the only one using `isNodeComingFromTestingLibrary` directly.